src/sentry/models/auditlogentry.py

```diff
@@ -87,6 +87,8 @@ class AuditLogEntryEvent(object):
 
     INTERNAL_INTEGRATION_ADD = 130
 
+    INTEGRATION_ADD_TOKEN = 135
```

Comment:
Errp sorry this is my bad in the comment I left but this should actually be `INTERNAL_INTEGRATION_ADD_TOKEN`  (and then `internal-integration.add-token`) since the `integration.add`, `integration.edit`, `integration.remove` all related to old plugins and not the integration platform)

Reply:
Also looks like we didn't add `INTERNAL_INTEGRATION_ADD` down below... hrm. You don't have to add that in this PR necessarily but we should make sure to add that at some point